<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>761</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>客户端信息：</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>客户端操作</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnct"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuQuery">
    <property name="title">
     <string>机票查询</string>
    </property>
    <addaction name="actionConditionQuery"/>
    <addaction name="actionAllQuery"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionShow"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuOperation">
    <property name="title">
     <string>机票购买</string>
    </property>
    <addaction name="actionBuyTicket"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuQuery"/>
   <addaction name="menuOperation"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnct"/>
   <addaction name="actionExit"/>
   <addaction name="actionConditionQuery"/>
   <addaction name="actionAllQuery"/>
   <addaction name="actionBuyTicket"/>
  </widget>
  <action name="actionConnect">
   <property name="icon">
    <iconset>
     <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>连接服务器</string>
   </property>
  </action>
  <action name="actionDisconnct">
   <property name="icon">
    <iconset>
     <normaloff>:/images/scissors.png</normaloff>:/images/scissors.png</iconset>
   </property>
   <property name="text">
    <string>断开连接</string>
   </property>
  </action>
  <action name="actionBuyTicket">
   <property name="icon">
    <iconset>
     <normaloff>:/images/folder.png</normaloff>:/images/folder.png</iconset>
   </property>
   <property name="text">
    <string>购买机票</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>:/images/power-off.png</normaloff>:/images/power-off.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionConditionQuery">
   <property name="icon">
    <iconset>
     <normaloff>:/images/download.png</normaloff>:/images/download.png</iconset>
   </property>
   <property name="text">
    <string>查询特定航班</string>
   </property>
  </action>
  <action name="actionAllQuery">
   <property name="icon">
    <iconset>
     <normaloff>:/images/download.png</normaloff>:/images/download.png</iconset>
   </property>
   <property name="text">
    <string>查询所有航班</string>
   </property>
  </action>
  <action name="actionShow">
   <property name="text">
    <string>显示内容</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>购买机票</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
